lib/MusicBrainz/Server/Entity/Role/Rating.pm

```diff
@@ -16,6 +16,18 @@ has 'rating_count' => (
     isa => 'Int'
 );
 
+around TO_JSON => sub {
+    my ($orig, $self) = @_;
+
+    my $json = $self->$orig;
+
+    $json->{rating} = $self->rating;
+    $json->{rating_count} = $self->rating_count;
```

Comment:
I would expect it to default to `0`.

Reply:
Fixed; also fixed the Flow types to account for `rating` or `user_rating` possibly being null.